core/include/cubos/core/log.hpp

```diff
@@ -375,6 +375,11 @@ namespace cubos::core
         static const char* streamFormat(memory::Stream& stream, const char* format, const reflection::Type& type,
                                         const void* value);
     };
+
+    /// @brief Cross-platform utility to get last error message.
+    /// @return Message.
+    /// @todo Move this to a higher module for general purposes?
+    std::string logSystemError();
```

Comment:
Hm, maybe you should rebase the UDP PR on this one and merge it only after this one is merged, as it would also benefit from this function.